src/extensions/spdlog/SpdlogToActs.h

```diff
@@ -4,38 +4,124 @@
 
 #pragma once
 
```

Comment:
Code should become MPL https://eic-code-browser.sdcc.bnl.gov/lxr/source/include/Acts/Utilities/Logger.hpp